docs/use-github-in-bcgov/bc-government-organizations-in-github.md

```diff
@@ -41,31 +40,110 @@ The `bcgov-c` organization stores temporary (up to 12 months), private repositor
+ 
+Ministries won’t be charged for upgraded GitHub licenses.  However, contractors need IDIRs before they can enable SSO. Ministries may need to seek expense approvals to order [IDIRs for contracted resources without one](https://citz-do.atlassian.net/servicedesk/customer/portal/2/article/78053380). 
+  
+### Directions to link your accounts before July 30
+ 
+#### For employees 
+ 
+[placeholder for video to show what to expect]()
+ 
+1. Have any two-factor devices, and personal GitHub login and IDIR login ready for linking 
+2. Log into GitHub and follow the prompts on the pop up to connect your GitHub and IDIR accounts together. [Learn more about how this works](). 
+3. Reauthorize your personal access tokens and/or SSH keys so they continue to work with SSO enabled. [Learn more about how to do this](). 
+ 
+That’s it, your accounts are now linked.  
+ 
+#### For contractors 
+ 
+If you don’t have an IDIR account, [we have guidance on ordering a compatible IDIR] ](https://citz-do.atlassian.net/servicedesk/customer/portal/2/article/78053380). It can take several days to complete depending on your ministry.We recommend doing with this step as soon as possible to avoid access disruptions.  
+ 
+After getting an IDIR, follow the steps below: 
+
+[placeholder for video to show what to expect]()
+
+1. Have any two-factor device, and personal GitHub login and IDIR login ready for linking 
+2. Log into GitHub and follow the prompts on the pop up to connect your GitHub and IDIR accounts together. [Learn more about how this works](). 
+3. If linking fails, your IDIR configuration is not compatible with our GitHub SSO. You must [request an upgrade to a P2 level IDIR account]().  
+4. Reauthorize your personal access tokens and/or SSH keys so they continue to work with SSO enabled. [Learn more about how to do this](). 
+ 
+That’s it, your accounts are now linked.  
+ 
+### Directions to link your account after July 30 
+
+If you are a contractor without an IDIR, follow the steps in the [IDIR transition guide](https://citz-do.atlassian.net/servicedesk/customer/portal/2/article/78053380) before proceeding. 
+ 
+1. If you have a valid IDIR, ask anyone with access to the organization to re-invite you. [Learn how to request access to the bcgov GitHub organization](https://developer.gov.bc.ca/docs/default/component/bc-developer-guide/use-github-in-bcgov/bc-government-organizations-in-github/#bcgov). 
+2. Once you have access again, your teammates with admin access may need to re-invite you to the repositories you had access to before.  
+3. Reauthorize your personal access tokens and/or SSH keys so they continue to work with SSO enabled. [Learn more about how to do this](). 
+ 
+### Upgrading other GitHub organizations 
+ 
+The bcgov-c, BCDevOps organizations will transition to SSO on different timelines. It will be the same process moving forward for any organization upgrade.  
+ 
+[Sign up to the DevX newsletter]() for updates on these changes. 
```

Comment:
@Shivanganii ah shoot, we need to track down how to make a sign up page for Mail Chimp - something for standup!